ably/realtime_conn_spec_test.go

```diff
@@ -2040,6 +2042,7 @@ func TestRealtimeConn_RTN14g(t *testing.T) {
 		change := make(ably.ConnStateChanges, 1)
 		c.Connection.OnAll(change.Receive)
 		var state ably.ConnectionStateChange
+		ablytest.Instantly.Recv(t, nil, change, t.Fatalf) // Skip CONNECTING
```

Comment:
This is racy. To ensure we'll get the CONNECTING state into the `change` channel, the `c.Connect` must be below the `OnAll` listener registration.

Reply:
done 80ccec5874864feea4babdcc1db112cf30fdee3e